<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordEditor</class>
 <widget class="QDialog" name="WordEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit word</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Sound:</string>
       </property>
       <property name="buddy">
        <cstring>sound</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="sound">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>S&amp;pelling:</string>
       </property>
       <property name="buddy">
        <cstring>spelling</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Type:</string>
       </property>
       <property name="buddy">
        <cstring>wordType</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="wordType">
       <item>
        <property name="text">
         <string>General Noun</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Name of People</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Geographic Name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Adjective</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Adverb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Single Kanji Character</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="generalNoun">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="nounNa">
           <property name="text">
            <string>-na</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="nounSa">
           <property name="text">
            <string>-sa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="nounSuru">
           <property name="text">
            <string>-suru</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="nounOnly">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="nounGa">
           <property name="text">
            <string>-ga</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="adjective">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="adjectiveTo">
           <property name="text">
            <string>-to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="adjectiveTaru">
           <property name="text">
            <string>-taru</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="adjectiveSuru">
           <property name="text">
            <string>-suru</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="adjectiveOnly">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Frequency:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="frequency">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="spelling">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sound</tabstop>
  <tabstop>spelling</tabstop>
  <tabstop>wordType</tabstop>
  <tabstop>nounNa</tabstop>
  <tabstop>nounSa</tabstop>
  <tabstop>nounSuru</tabstop>
  <tabstop>nounOnly</tabstop>
  <tabstop>nounGa</tabstop>
  <tabstop>adjectiveTo</tabstop>
  <tabstop>adjectiveTaru</tabstop>
  <tabstop>adjectiveSuru</tabstop>
  <tabstop>adjectiveOnly</tabstop>
  <tabstop>frequency</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WordEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WordEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
